### PR TITLE
VZ 4355: Ensure namespace deletion during tests

### PR DESCRIPTION
**Pull request by anders-swanson**

### CODE REVIEW

File: tests/e2e/pkg/kubernetes.go [hunk 1]
```diff
@@ -521,6 +521,16 @@ func CreateNamespace(name string, labels map[string]string) (*corev1.Namespace,
 	return ns, nil
 }
 
+func RemoveNamespaceFinalizers(namespace *corev1.Namespace) error {
+	clientset, err := k8sutil.GetKubernetesClientset()
+	if err != nil {
+		return err
+	}
+	namespace.ObjectMeta.Finalizers = nil
```
Comment (pfmackin) [inline tests/e2e/pkg/kubernetes.go:529]: won't this remove annotations?

Comment (anders-swanson) [reply tests/e2e/pkg/kubernetes.go:529]: let me check. although it may not matter since we are using this in the context of cleaning up a namespace

Comment (pfmackin) [reply tests/e2e/pkg/kubernetes.go:529]: yeah if you are deleting the NS then it doesn't matter

Comment (anders-swanson) [reply tests/e2e/pkg/kubernetes.go:529]: annotations are not removed

**Review by pfmackin — APPROVED** [review]

**Pull request merged**